docs/source/develop.rst

```diff
@@ -95,7 +95,9 @@ Install Dask and dependencies::
 
 For development, Dask uses the following additional dependencies::
 
-   pip install pytest moto mock
+   pip install pytest moto mock ipython graphviz
+
+You will also need graphviz installed on your system.
```

Comment:
There is text at the bottom of http://docs.dask.org/en/latest/graphviz.html which you may want to copy or refer to

Reply:
After #4720 the tests pass with just:

```
cd dask
pip install -e ".[complete]"
pip install pytest
py.test dask
```

I added the required `pytest.importorskip` lines there to remove these optional dependencies.

Reply:
This is much better than adding more instructions. Thanks @jcrist.